tests/my_house_tests.py

```diff
@@ -92,6 +102,26 @@ def test_not_add_expenses_to_my_house_with_bad_description(self, client, new_use
         assert len(Expenses.objects.all()) == 0
         assert response.status_code == 200
 
+    def test_edit_expenses_of_my_house(self, client, new_user):
+        client.force_login(new_user)
+        house = HouseFactory(user=new_user)
+        house.save()
+        ExpenseFactory(house=house, month=1).save()
+        response = client.post('/house/edit_expense/1/', OTHER_EXPENSE_FORM_DATA)
+        assert len(Expenses.objects.all()) == 1
+        assert response.status_code == 302
+        assert response.url == '/../house'
+
```

Comment:
You should also assert that the expense was actually changed

Reply:
fixed